src/main/java/uk/ac/ebi/ampt2d/accession/file/UuidFileAccessionRepository.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.file;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Collection;
+import java.util.UUID;
+
+@Repository
```

Comment:
Setting a profile in this class was deliberate, in order to inject the repositories depending on the profile selected (for instance, UUID vs sequential). It is true that it could have been simplified to just "uuid" instead of having entity-based granularity.

This comment applies to other classes.